### PR TITLE
Migration fix for new query type

### PR DESCRIPTION
**Pull request by briangann**
This PR is to fix the panel migrations for the non-query created clock panels.  It will also cleanup the (angular) configs if they are found outside of the config block.

Other modifications:
- Forces yarn V1
- Updates to node 20
- Bumps min Grafana version to 8.3+ (was 8.0)

TODO:
- [x] Migrates old panels and cleans up
- [x] Updates create-plugin changes
- [x] Adds basic playwright config (simple setup)

### CODE REVIEW

File: src/plugin.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "updated": "%TODAY%"
   },
   "dependencies": {
-    "grafanaDependency": ">=8.0.0",
+    "grafanaDependency": ">=10.0.0",
```
[the remark this answers is not in the document]
Comment (briangann) [reply src/plugin.json]: I've adjust the dependencies as much as possible, and the oldest version that will work is

Prior to version 8.3.11, I get this error when using anything inside the panel editor:

(tested 8.0.8, 8.1.8, 8.2.7)
TypeError: Cannot read properties of undefined (reading 'mode')

8.3.x and higher work fine now.

**Comment by briangann** [comment]
almost ready with this one, the migration tests ironed out some issues, but still need to test some real instances with dashboards that are throwing the error, and also with query-enabled clocks.

**Comment by briangann** [comment]
we'll need to update the docs in another PR, would like to get this fix out the door asap.

**Review by tolzhabayev — APPROVED** [review]

**Comment by briangann** [comment]
Fixes #189 
Fixes #186

**Pull request merged**